ADOPTERS.md

```diff
@@ -1,5 +1,7 @@
 # Adopters
  This is a list of production adopters of Banzai Cloud's Kafka Operator (in alphabetical order):
+ 
+- Adobe Digital Marketing group is using the Kafka operator with the Zookeeper operator.
```

Comment:
@seba I suggest the following wording:

Adobe Experience Platform is using the Kafka Operator to enable consistent deployment, dynamic scaling, smooth upgrades and auto-balancing in our cross-cloud Kafka infrastructure. Adobe is also contributing to the open source version of the operator.


Thanks